lib/node_modules/@stdlib/stats/base/ndarray/dmeanstdev/docs/repl.txt

```diff
@@ -0,0 +1,43 @@
+
+{{alias}}( arrays )
+    Computes the mean and standard deviation of a one-dimensional double-
+    precision floating-point ndarray.
+
+    If provided an empty ndarray, the function returns `NaN` values.
+
+    Parameters
+    ----------
+    arrays: ArrayLikeObject<ndarray>
+        Array-like object containing the following ndarrays in order:
+
+        - a one-dimensional input ndarray.
+        - a one-dimensional output ndarray (of length 2) to store the mean
+        and standard deviation.
+        - a zero-dimensional ndarray specifying the degrees of freedom
+        adjustment.
+
+    Returns
+    -------
+    out: ndarray
+        An ndarray containing the mean and standard deviation.
+
+    Examples
+    --------
+    > var xbuf = new {{alias:@stdlib/array/float64}}( [ 2.0, 1.0, 2.0, -2.0 ] );
+    > var dt = 'float64';
+    > var sh = [ xbuf.length ];
+    > var sx = [ 1 ];
+    > var ox = 0;
+    > var ord = 'row-major';
+    > var x = new {{alias:@stdlib/ndarray/ctor}}( dt, xbuf, sh, sx, ox, ord );
+    > var o = new {{alias:@stdlib/array/float64}}( 2 );
+    > var out = new {{alias:@stdlib/ndarray/ctor}}( dt, o, [ 2 ], [ 1 ], ox, ord );
+    > var opts = { 'dtype': dt };
+    > var correction = {{alias:@stdlib/ndarray/from-scalar}}( 1.0, opts );
+    > {{alias}}( [ x, out, correction ] );
+    > {{alias:@stdlib/ndarray/to-array}}( out )
```

Comment:
This is precisely the sort of thing you can do in that README example. Namely, use `ndarray/to-array` to convert the output ndarray to a nested array and then show approximate results as done on the next line.